episodes/introduction.md

```diff
@@ -89,10 +179,24 @@ All of the above are things a Hub Champion should do to enable their community t
 
```

Comment:
Add something here about how a hub can address the challenges that the learners' communities face?

Reply:
I've added this as an extra question in bb5081fa1ea6d998a156e2334e0b9a6a27cc8ac1